### DISCUSSION TITLE
Cyberduck Connection

### DISCUSSION THREAD

**Discussion by usmcamgrimm** [iterate-ch/cyberduck#15786] [Q&A]
Trying to connect with WebDav but the URL is appending my username. Is there a way to add my username without adding it to the connection URL?

**Comment by tatze96** [comment]
It is expected and intended that the username and, if set, the path gets added to the connection URL. 

As long as the *Server* field only contains the hostname and the *Username* and *Password* fields have added the corresponding credentials, the connection to the server should work without issues.
